guake/paths.py.in

```diff
@@ -94,3 +96,5 @@ AUTOSTART_FOLDER = {{ AUTOSTART_FOLDER }}
 def try_to_compile_glib_schemas():
     log.info("Compiling schema: %s", SCHEMA_DIR)
     subprocess.check_call(["glib-compile-schemas", "--strict", SCHEMA_DIR])
+    with open(Path(SCHEMA_DIR, "version"), "w", encoding="ascii") as version:
+        version.write(guake.guake_version())
```

Comment:
I think you will have to use the same file name + extension to avoid any conflict